.travis.yml

```diff
@@ -9,6 +9,8 @@ before_install:
   - wget 'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-linux-x86_64.tar.bz2'
   - tar xf phantomjs-1.9.7-linux-x86_64.tar.bz2
   - export PATH=$PATH:phantomjs-1.9.7-linux-x86_64/bin/
+  - psql -c "CREATE USER accounts WITH SUPERUSER PASSWORD 'accounts'" -U postgres
```

Comment:
@karenc - Looks good. 

It'd be nice if we can remove the create db requirement at a later point.   Instead the tests can do drop all on the tables and recreate them.